package.json

```diff
@@ -63,6 +64,10 @@
     "*.js": [
       "eslint --fix",
       "git add"
+    ],
+    "examples/**/*.html": [
+      "npm run reference-tables",
+      "git add examples/index.html"
```

Comment:
this hook is cool, but I was testing it and it doesn't run when you are deleting a file, which might make it a bit of an inconsistent experience. I also think it's a bit weird to have something be automatically added to your commit -- maybe we should just stick the the failure in github checks letting you know you haven't regenerated that particular file?

Reply:
I'm good with dropping this if you want